doc/src/cheatsheet.md

```diff
@@ -0,0 +1,23 @@
+# Cheatsheet
+
+Users of Lalrpop has compiled the following cheatsheet table as a quick way to
+look up useful Lalrpop-isms. If you are looking for a specific piece of
+functionality, use this table to jump to the right section.
+
+| name | snipped | description | tutorial |
+|---|---|---|---|
+| position | `<@L> T <@R>` | uhm, I think it gives the index of the parser in the input | - |
+| error_recovery | `! => { ... }` | recover from parser errors | [Error recovery](tutorial/008_error_recovery.md) |
+| grammar_parameter | `grammar(scale: isize);` | input parameters usable in the generated parser | [Passing state parameter](tutorial/009_state_parameter.md) |
+| ?? | `Num => func(<i32>)` | maybe automatic number parsing?? | - |
```

Comment:
What does this one do, exactly? Do you have a complete example? Because I'm not sure to know the feature you're referring to

Reply:
Yeah, I'm not sure what the original poster meant by this the more I look into it. The closest I was able to find was something like:
```
extern {
    enum Tok<'input> {
        "(" => Tok::LParen,
        ")" => Tok::RParen,
        "-" => Tok::Minus,
        "+" => Tok::Plus,
        "*" => Tok::Times,
        "/" => Tok::Div,
        Num => Tok::Num(<i32>),
        Fraction => Tok::Fraction(<i32>, <i32>), // Regression test for #179
    }
}
```

Reply:
Oh, I see. I believe it's the usual way to map terminals to lexer tokens when using a custom lexer. You have to specify the type of data that the token carries (and that you want to extract in the parser): this just means that `Fraction` is defined as `enum Tok { ..., Fraction(i32, i32), ... }` in the lexer, which will correspond to the non-terminal `Fraction` that will have type `(i32, i32)` in the parser.